lib/node_modules/@stdlib/blas/base/srotg/lib/assign.js

```diff
@@ -0,0 +1,95 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var abs = require( '@stdlib/math/base/special/abs' );
+var sqrt = require( '@stdlib/math/base/special/sqrt' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var copysign = require( '@stdlib/math/base/special/copysign' );
+
+
+// MAIN //
+
+/**
+* Construct a Givens plane rotation.
```

Comment:
```suggestion
* Constructs a Givens plane rotation.
```

For the actual function, we use the active voice. E.g., `Constructs`, rather than `Construct`. For the module and package description, we use the imperative voice. E.g., `Construct`.